packages/gestalt/src/Button.js

```diff
@@ -122,9 +139,11 @@ const ButtonWithForwardRef: React$AbstractComponent<
       [styles.selected]: !disabled && selected,
       [styles.disabled]: disabled,
       [styles.enabled]: !disabled,
```

Comment:
@AlbertCarreras looks like a disabled `<Button href="" disabled />` does not work as expected:

* The button (link) is still clickable
* It still opens the link

![image](https://user-images.githubusercontent.com/127199/91911686-73181980-ec66-11ea-9725-bb1d28e48e4c.png)

Reply:
I added _disable prop to Link so I can do `href={ _disable ? undefined : href}`

We can use this approach for id, role and, accessibilitySelected so we know they are private props and therefore not documented

Reply:
DONE